android/src/main/kotlin/live/hms/hmssdk_flutter/HMSLogSettings.kt

```diff
@@ -0,0 +1,26 @@
+package live.hms.hmssdk_flutter
+
+import live.hms.video.media.settings.HMSLogSettings
+import live.hms.video.utils.HMSLogger
+
+class HMSLogSettings {
+
+    companion object{
+
+        fun setLogSettings(maxDirSizeInBytes: Double?,logStorageEnabled: Boolean?,logLevel: String?): HMSLogSettings {
+            return  HMSLogSettings(maxDirSizeInBytes = maxDirSizeInBytes!!.toLong(), isLogStorageEnabled = logStorageEnabled!!, level = getLogLevel(logLevel))
```

Comment:
`maxDirSizeInBytes` and other parameters shouldn't be null or we should add a null check before creating the `HMSLogSettings` object.